pages/solidity-api/ChipRegistry.md

```diff
@@ -2,10 +2,11 @@
 
 ## ChipRegistry
 
+// Is it always Arx Protocol? is Arx Protocol the Manufacturer in this situation?
 Entrypoint for resolving chips added to Arx Protocol. TSMs can enroll new projects into this registry by specifying a
 ProjectRegistrar to manage chip claims. Chip claims are forwarded from ProjectRegistrars at which point a ERC-721
 compliant "token" of the chip is minted to the claimant and other metadata associated with the chip is set. Any project
-looking to integrate ERS chips should get resolution information about chips from this address. Because chips are
+looking to integrate ERS chips should get resolution information (TODO: any example of what resolution information might contain?) about chips from this address. Because chips are
 represented as tokens any physical chip transfers should also be completed on-chain in order to get full functionality
```

Comment:
as it says: any example of what resolution information might contain? just metadata, etc? assuming this is customizable